### PR TITLE
review-feature/footer

### PR DESCRIPTION
**Pull request by elawrobel**
And footer is done!
I would appreciate if you could take a look on it.

Best, 
E.

### CODE REVIEW

File: app/components/A/index.js [hunk 1]
```diff
@@ -6,9 +6,22 @@ import styled from 'styled-components';
 
 const A = styled.a`
   color: ${props => props.theme.colors.black};
+  font-family: ${props => props.theme.fonts.text};
```
Comment (ucanthurtme) [inline app/components/A/index.js:9]: 👍

File: app/components/Footer/index.js [hunk 2]
```diff
@@ -2,34 +2,89 @@ import React from 'react';
...
+  padding: 2em;
   border-top: 1px solid #666;
+  position: relative;
+
+  @media (min-width: 768px) {
+    padding: 2em 4em;
+  }
+`;
+
+const FcbLink = styled(Link)`
```
Comment (ucanthurtme) [inline app/components/Footer/index.js:25]: what does it mean fcb ?

File: app/components/Footer/index.js [hunk 3]
```diff
@@ -2,34 +2,89 @@ import React from 'react';
...
-          values={{
-            author: <A href="https://twitter.com/mxstbr">Max Stoiber</A>,
-          }}
-        />
-      </section>
+      <Img src={Logo} alt="Logo" />
+      <A to="/contact">
+        <FormattedMessage {...messages.contactMessage} />
+      </A>
+      <FcbLink to="https://www.facebook.com/groups/235355313496727/">
```
Comment (ucanthurtme) [inline app/components/Footer/index.js:84]: as a good practise => let's move it to some kind of const/config

**Comment by ucanthurtme** [comment]
ping me pls, when it'll be done then I'll try to run it locally

File: app/components/Footer/index.js (hunk 2 above)
Comment (elawrobel) [reply app/components/Footer/index.js:25]: Facebook Link. This is in footer and it'a a link with url to facebook group. It isn't ok?

**Comment by elawrobel** [comment]
@ucanthurtme it fixed! Would be perfect if you could take a look on it.

**Pull request merged**